rest_assured/testcases.py

```diff
@@ -215,6 +215,8 @@ class CreateAPITestCaseMixin(object):
     create_data = None
     #: The name of the field in the response data for looking up the created object in DB.
     response_lookup_field = 'id'
+    # set specific format for request data. Available values are multipart and json
+    format = 'json'
```

Comment:
@ydaniv updated. Now format could be configured for create and update independently